src/webapp/src/client/app/common/expandable-menu/expandable-menu.component.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('simoonaApp.Common')
+        .component('aceExpandableMenu', {
+            replace: true,
+            transclude: true,
+            bindings: {
+                menuName: '<',
+                menuIconClass: '<',
+                menuTextTranslation: '<',
+                isExpanded: '<'
+            },
+            templateUrl: 'app/common/expandable-menu/expandable-menu.html',
+            controller: expandableMenuController,
+            controllerAs: 'vm'
+        });
+
+    function expandableMenuController() {
+        let vm = this;
+
+        vm.onExpandClick = onExpandClick;
+
+        const wasExpanded = localStorage.getItem(vm.menuName);
```

Comment:
It would be good to add some prefix for the localstorage item to make it more clear when you would look at/debug localstorage. (And as general practice - menu names are quite generic, so names could conflict with other entries.)

Reply:
Added a postfix. If needed can change it to prefix.